app/controllers/AmbiguousJourneyController.scala

```diff
@@ -112,35 +112,35 @@ class AmbiguousJourneyController @Inject() (
 
   def processFiledReturnByPostChoice: Action[AnyContent] = VerifiedAction(baseBreadcrumb) {
     implicit pertaxContext =>
-      AmbiguousUserFlowDto.form.bindFromRequest.fold(
-        formWithErrors => {
-          Future.successful(BadRequest(views.html.ambiguousjourney.filedReturnByPostChoice(formWithErrors)))
-        },
-        ambiguousFiledOnlineChoiceDto => {
-          ambiguousFiledOnlineChoiceDto.value match {
-            case true => Future.successful(Redirect(routes.AmbiguousJourneyController.usedUtrToRegisterChoice()))
-            case false => {
-              if (configDecorator.saAmbigSkipUTRLetterEnabled)
-                Future.successful(Redirect(routes.AmbiguousJourneyController.usedUtrToEnrolChoice()))
-              else
-                Future.successful(Redirect(routes.AmbiguousJourneyController.receivedUtrLetterChoice()))
-            }
+    AmbiguousUserFlowDto.form.bindFromRequest.fold(
```

Comment:
Only change here is the tabbing, can you undo this change

Reply:
done